src/components/DocumentationTopic/PrimaryContent/DeclarationGroup.vue

```diff
@@ -102,8 +195,48 @@ export default {
   }
 }
 
+.declaration-pill--expanded {
+  transition-timing-function: linear;
+  transition-property: opacity, height;
+  $docs-declaration-source-border-width: 1px;
+
+  .source {
+    border-width: $docs-declaration-source-border-width;
+
+    // ensure links are not clickable, when expanded
+    :deep(a) {
+      pointer-events: none;
```

Comment:
Which scenario are we trying to prevent links from being clickable? Just curious

Reply:
When the whole thing is expanded, you could by accident click on a link to a symbol or something, in the declaration. This prevents that, until you collapse the declarations.

Reply:
Oh, got it—we want clicking the a declaration in the expanded list to switch the page and then have the normal clicking links within the selected declaration. That makes sense, thanks.